app/slack_bot.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from dotenv import load_dotenv
+import os
+load_dotenv()
+
+def slack_message(message):
```

Comment:
👍  love to see that you separated this out into its own file and helper function